src/utils.js

```diff
@@ -0,0 +1,7 @@
+export function getApiUrl() {
+  return window.localStorage.getItem('API_URL') ?? process.env.API_URL;
+}
+
+export function getTokensFromLocalStorage() {
+  return JSON.parse(window.localStorage.getItem('tokens') ?? '{}');
+}
```

Comment:
```suggestion
export function getToken({walletAddress}) {
  try {
    const tokens = window.localStorage.getItem('tokens') 
      ? JSON.parse(window.localStorage.getItem('tokens')) 
      : {}; 
    return tokens[walletAddress];  
  } catch (e) {
    return undefined;
  }
}
```
No need to know tokens for other wallet addresses.

Reply:
Added `restoreToken`